src/OpenTelemetry/Transport/OtlpHttpTransportFactory.php

```diff
@@ -26,12 +26,12 @@ public function supports(#[\SensitiveParameter] ExporterEndpointInterface $endpo
     public function createTransport(#[\SensitiveParameter] ExporterEndpointInterface $endpoint, ExporterOptionsInterface $options): TransportInterface
     {
         $params = $options->toTransportParams();
-        $format = OtlpExporterFormatEnum::tryFrom($params->contentType) ?? OtlpExporterFormatEnum::Json;
```

Comment:
`OtlpExporterFormatEnum::tryFrom($params->contentType)` always null because enum contains `format` but `$params->contentType` contains `application/format`.